### PR TITLE
Paul another bunch of missed endpoints

### PR DESCRIPTION
**Pull request by pisaychuk**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -19703,6 +19703,425 @@
...
+      }
+    },
+    "/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/totalSeats": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the total number of seats on sip peer location.",
+        "operationId": "GetSipPeerTotalSeats",
+        "summary": "Get Sip Peer Total Number of Seats.",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:19941]:
```suggestion
        "summary": "Get Sip Peer Total Number of Seats",
```

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -19703,6 +19703,425 @@
...
+      }
+    },
+    "/accounts/{accountId}/sites/{siteId}/totalSeats": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the total number of seats on site.",
+        "operationId": "GetSiteTotalSeats",
+        "summary": "Get Site Total Number of Seats",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:20010]:
```suggestion
        "summary": "Get Total Number of Seats for Site",
```

File: site/specs-temp/numbers.json [hunk 3]
```diff
@@ -19703,6 +19703,425 @@
...
+      }
+    },
+    "/accounts/{accountId}/totalSeats": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the total number of seats on account.",
+        "operationId": "GetAccountTotalSeats",
+        "summary": "Get Account Total Number of Seats",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:20076]:
```suggestion
        "summary": "Get Total Number of Seats for Account",
```

**Review by ajrice6713 — DISMISSED** [review]

**Comment by pisaychuk** [comment]
@brianluisgomez hey! I know that Michael is on PTO this week. Can you pls take a look and merge if no questions, I already have an approve from AJ. Thanks!

File: site/specs-temp/numbers.json [hunk 4]
```diff
@@ -19706,6 +19706,425 @@
           }
         }
       }
+    },
+    "/accounts/{accountId}/{orderType}/{orderId}/adminparameters": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Get order admin parameters.",
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:19715]: Can you remove all the `.` in the one sentence descriptions?

File: site/specs-temp/numbers.json [hunk 5]
```diff
@@ -19706,6 +19706,425 @@
           }
         }
       }
+    },
+    "/accounts/{accountId}/{orderType}/{orderId}/adminparameters": {
+      "get": {
+        "tags": [
+          "/accounts"
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:19713]: We've been trying to move away from `/accounts`?  Is there a better tag we could use for these?  Maybe `Orders`?

File: site/specs-temp/numbers.json (hunk 1 above)
Comment (mrmeyers99) [reply site/specs-temp/numbers.json:19941]: I don't think you addressed this comment yet right?

Comment (pisaychuk) [reply site/specs-temp/numbers.json:19941]: Sorry, Michael, thought I done with this :/

**Review by mrmeyers99 — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**